machine/machine_core/machine.py

```diff
@@ -152,7 +153,7 @@ def journal_messages(self, matches, log_level, cursor=None):
         self.execute("journalctl --sync 2>/dev/null || true; sleep 1; journalctl --sync 2>/dev/null || true")
 
         # Prepend "SYSLOG_IDENTIFIER=" as a default field, for backwards compatibility
-        matches = (m if re.match("[a-zA-Z0-9_]+=", m) else "SYSLOG_IDENTIFIER=" + m for m in matches)
+        all_matches = (m if re.match("[a-zA-Z0-9_]+=", m) else "SYSLOG_IDENTIFIER=" + m for m in matches)
```

Comment:
Nag about existing code: I don't like capturing iterators into variables like this — we've been bit by things like this before.  Either turn this into a list comprehension or pull the `" + " .join()` up here.

Doing it as a list comprehension would actually allow you to avoid the variable rename, since it's the same type...